lua/lvim/core/which-key.lua

```diff
@@ -176,7 +176,6 @@ M.config = function()
       -- " Debugging
       g = {
         name = "Git",
-        g = { "<cmd>lua require 'lvim.core.terminal'.lazygit_toggle()<cr>", "Lazygit" },
         j = { "<cmd>lua require 'gitsigns'.next_hunk({navigation_message = false})<cr>", "Next Hunk" },
         k = { "<cmd>lua require 'gitsigns'.prev_hunk({navigation_message = false})<cr>", "Prev Hunk" },
```

Comment:
I was actually thinking that `[c` and `]c` instead for these. It would also be nice to have a faster keybind for a such common operation.

Reply:
it looked like you added those lines 😅